### DISCUSSION TITLE
No messages to submit using default /commit feature

### DISCUSSION THREAD

**Discussion by BuilderZac** [olimorris/codecompanion.nvim#177] [Q&A] [answered]
This is a really neat plugin and I am trying to add it to my work flow but I am having some problems. Don't know if this is just user error which is why I am not opening a bug report yet. Is there anything specific thing you have to do for it to write up a commit message? I am using the commit option from the :CodeCompanionActions list but it keeps giving me warnings that there is no message to submit but all the data in the chat buffer correctly generated. Also even if I type more in the buffer and do ctrl+s to manually send it still does not work.

**Comment by olimorris** [comment]
I suspect this is a Tree-sitter issue. I've just updated the health check to include this. Update the plugin and run `:checkhealth codecompanion`

You can try using the `minimal.lua` file from the bug report to test.

**Reply by BuilderZac** [reply]
Thanks for getting back so soon. Just pulled the update and I pass all checks but the error persists. Will start getting opening a bug report and load up minimal.lua.

**Reply by BuilderZac** [reply]
I cant recreate it with the minimal.lua config. Looks like a problem on my end. The way I have it set up in my main config is basicaly a copy of this config https://github.com/olimorris/codecompanion.nvim/blob/main/lua/codecompanion/config.lua with my adapters set up. Also in the log no data is generated when this occurs.

**Reply by BuilderZac** [reply, marked as answer]
Okay I re-pulled the https://github.com/olimorris/codecompanion.nvim/blob/main/lua/codecompanion/config.lua and redid my adapter changes and now it works. Looks like I may have gotten a bad copy or something went wrong the first time.

**Reply by olimorris** [reply]
Have you been able to get it working?

**Reply by BuilderZac** [reply]
Since re-pulling the default config, it's been working. It's a really neat system you got set up good job on it. It should become a massive part of my debugging and code revision process.
